sp.yml

```diff
@@ -0,0 +1,39 @@
+---
+- hosts: "{{ target | default('localhost') }}"
+  become: no
+  connection: ssh
+  gather_facts: true
+
+  vars_files:
+    - "inventory/host_vars/all"
+    - "inventory/group_vars/all"
+    - "inventory/group_vars/{{ env }}/vars.yml"
+    - "inventory/group_vars/{{ env }}/vault.yml"
+    - "inventory/group_vars/{{ pop }}/vars.yml"
+    - "inventory/group_vars/{{ pop }}/vault.yml"
+    - "inventory/group_vars/{{ plat }}/vars.yml"
+    - "inventory/group_vars/{{ plat }}/vault.yml"
+
+  pre_tasks:
+    - name: get my public IP
+      ipify_facts:
+      #validate_certs: no
+
+  roles:
+    - { role: update-os-pckgs, become: true }
```

Comment:
I would skip this role, we don't want to update the os each time we run the SP. right?

Reply:
right for you but not for me: we are selling CI/CD but we don't want to eat our own food.
this enforcement shows early detected issues, namely on DEV, PRE-INT, INT, QUAL, etc, stages.
ex, every time Ansible does a major update, new warnings appear on well behaved plays in the past: as soon as it is detected, the better